components/serializer.rst

```diff
@@ -162,6 +139,149 @@ needs three parameters:
 2. The name of the class this information will be decoded to
 3. The encoder used to convert that information into an array
 
+Attributes Groups
+-----------------
+
+.. versionadded:: 2.7
+    The support of serialization and deserialization groups was introduced
+    in Symfony 2.7.
+
+Sometimes, you want to serialize different set of attributes from your
+entities. Groups are a handy way to achieve this need.
+
+Assume you have the following object plain-old-PHP object::
+
+    namespace Acme;
+
+    class MyObj
+    {
+        public $foo;
+        public $bar;
+    }
+
+The definition of serialization can be specified using annotations, XML
+or YAML. The :class:`Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactory`
```

Comment:
sorry, didn't think about this the previous time: you have to escape all backslashes when used inside roles. So ``:class:`Symfony\\Component\\Serializer\\Mapping\\Factory\\ClassMetadataFactory` `` (same applies below)